src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9729,15 +9713,23 @@ private WhenClauseSyntax ParseWhenClause(Precedence precedence)
         }
 
         /// <summary>
-        /// Parse a local variable declaration.
+        /// Parse a local variable declaration.  Specifically, only for the `fixed (...)` `for(...)` or `using (...)`
+        /// statements.  Specifically, the statements where a close paren comes after the variable declaration.
         /// </summary>
         /// <returns></returns>
-        private VariableDeclarationSyntax ParseVariableDeclaration()
+        private VariableDeclarationSyntax ParseVariableDeclaration_FixedForUsingStatement()
```

Comment:
Consider using a name like `ParseParenthesizedVariableDeclaration`

Reply:
that's a much better name.

Reply:
done.